### PR TITLE
add the patched images for volume map & base image cve patches.

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -427,6 +427,7 @@ PATCHED_HYPERKUBE_IMAGES="
 1.14.8_f0.0.4
 1.14.8-hotfix.20200326
 1.14.8-hotfix.20200408.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:429]: This was just added, do we need it?

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -437,17 +438,19 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.10-hotfix.20200408.1
 1.15.11
 1.15.11.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:440]: do we need 1.15.11.1?

File: vhdbuilder/packer/install-dependencies.sh [hunk 3]
```diff
@@ -437,17 +438,19 @@ PATCHED_HYPERKUBE_IMAGES="
...
+1.15.11-hotfix.20200529.1
 1.16.0_v0.0.5
 1.16.7-hotfix.20200326
 1.16.7-hotfix.20200408
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:444]: This

File: vhdbuilder/packer/install-dependencies.sh [hunk 4]
```diff
@@ -437,17 +438,19 @@ PATCHED_HYPERKUBE_IMAGES="
...
-1.16.7-hotfix.20200408.1
+1.16.7-hotfix.20200601.1
 1.16.8
 1.16.8.1
 1.16.9
 1.16.9.1
+1.16.9-hotfix.20200529.1
 1.17.3-hotfix.20200326
 1.17.3-hotfix.20200408
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:452]: this

File: vhdbuilder/packer/install-dependencies.sh [hunk 5]
```diff
@@ -437,17 +438,19 @@ PATCHED_HYPERKUBE_IMAGES="
...
+1.15.11-hotfix.20200529.1
 1.16.0_v0.0.5
 1.16.7-hotfix.20200326
 1.16.7-hotfix.20200408
-1.16.7-hotfix.20200408.1
+1.16.7-hotfix.20200601.1
 1.16.8
 1.16.8.1
 1.16.9
 1.16.9.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:449]: this

File: vhdbuilder/packer/install-dependencies.sh (hunk 4 above)
Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:452]: https://github.com/Azure/AgentBaker/pull/75/files

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (andyliuliming) [reply vhdbuilder/packer/install-dependencies.sh]: removed 
1.14.8-hotfix.20200408.1

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh]: yes, this one was just added , and can be removed

File: vhdbuilder/packer/install-dependencies.sh (hunk 2 above)
Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh]: this is also just added by Zhongyi

File: vhdbuilder/packer/install-dependencies.sh (hunk 3 above)
Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:442]: this is still in use

File: vhdbuilder/packer/install-dependencies.sh (hunk 5 above)
Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:447]: this is just added by Zhongyi

File: vhdbuilder/packer/install-dependencies.sh (hunk 4 above)
Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:450]: this is in use.

File: vhdbuilder/packer/install-dependencies.sh [hunk 6]
```diff
@@ -388,18 +388,18 @@ K8S_VERSIONS="
...
 1.15.10-hotfix.20200408.1
-1.15.11.1
+1.15.11-hotfix.20200529.1
 1.16.0_v0.0.5
-1.16.7-hotfix.20200408.1
+1.16.7-hotfix.20200601.1
 1.16.8.1
 1.16.9.1
-1.17.3-hotfix.20200408.1
+1.17.3-hotfix.20200601.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:402]: 1.17.3 is not supported, do we still need to patch?

File: vhdbuilder/packer/install-dependencies.sh [hunk 7]
```diff
@@ -388,18 +388,18 @@ K8S_VERSIONS="
...
 1.15.3_v0.0.5
 1.15.4_v0.0.5
 1.15.5_f0.0.2
 1.15.7-hotfix.20200326
 1.15.10-hotfix.20200408.1
-1.15.11.1
+1.15.11-hotfix.20200529.1
 1.16.0_v0.0.5
-1.16.7-hotfix.20200408.1
+1.16.7-hotfix.20200601.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:399]: 1.16.7 is not supported

File: vhdbuilder/packer/install-dependencies.sh [hunk 8]
```diff
@@ -388,18 +388,18 @@ K8S_VERSIONS="
...
+1.14.8-hotfix.20200529.1
 1.15.3_v0.0.5
 1.15.4_v0.0.5
 1.15.5_f0.0.2
 1.15.7-hotfix.20200326
 1.15.10-hotfix.20200408.1
-1.15.11.1
+1.15.11-hotfix.20200529.1
 1.16.0_v0.0.5
-1.16.7-hotfix.20200408.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:399]: why do we change this?

**Pull request merged**